design/eti/override/templates/full/dichiarazioni.tpl

```diff
@@ -60,9 +60,8 @@
 	{rdelim};
 	var mainQuery = "{concat( 'classes [dichiarazione_impresa]', $ente_filter_string|wash(javascript))}";
     var geoMainQuery = 'classes [sede_azienda]';
+    var SessionCacheKey = "{concat($user.contentobject_id,'_dichiarazioni_impresa')}";
```

Comment:
Questo scassava la michia quando cambiavi utente pertrché teneva in memoria la query dell'utente precedente